paper/src/main/java/vg/civcraft/mc/citadel/listener/BlockListener.java

```diff
@@ -559,7 +559,7 @@ private void INTERNAL_checkLecternModificationPermission(final Cancellable event
 	public void onMossSpread(BlockFertilizeEvent event) {
 		for (BlockState block : event.getBlocks()) {
 			if (Citadel.getInstance().getReinforcementManager().getReinforcement(block.getBlock()) != null) {
-				event.getPlayer().sendMessage(Component.text("You can't do that while their are reinforced blocks around!").color(NamedTextColor.RED));
+				event.getPlayer().sendMessage(Component.text("You can't do that while there are reinforced blocks around!").color(NamedTextColor.RED));
```

Comment:
```suggestion
				event.getPlayer().sendMessage(Component.text("You can't bonemeal moss blocks while there are reinforced blocks nearby!").color(NamedTextColor.RED));
```
Alternatively, I feel this might be a little more clear of a message too, removing the pronouns for nouns

Reply:
Hmm only this is specifically locking into moss blocks, very recently learned this event also fires for nylium

Reply:
Then the pronouns (in the actual PR diff) is probably the way